source/guides/stacks/dlrs.rst

```diff
@@ -651,20 +653,20 @@ In this example, you will:
       --reverse_input_channels
 
 
-   You should now see three files in your working directory, :file:`frozen_inference_graph.bin`, :file:`frozen_inference_graph.mapping`, and :file:`frozen_inference_graph.xml`. These are your new models in the Intermediate Representation (IR) format and they are ready for use in the OpenVINO Inference Engine.
+   You should now see three files in your working directory, :file:`frozen_inference_graph.bin`, :file:`frozen_inference_graph.mapping`, and :file:`frozen_inference_graph.xml`. These are your new models in the Intermediate Representation (IR) format and they are ready for use in the OpenVINO™ Inference Engine.
 
 
 
-Using the OpenVino Inference Engine
-***********************************
+Using the OpenVINO™ toolkit Inference Engine
+********************************************
 
 This example walks through the basic instructions for using the inference engine.
 
 #. Starting the Model Server
 
    The process is similar to how we start `Jupter notebooks` on our containers
 
-   Run this command to spin up a OpenVino model fetched from GCP
+   Run this command to spin up a OpenVINO™ toolkit model fetched from GCP
 
    .. code-block:: bash
```

Comment:
@bd-dean  the model storage path has been changed, let's update the model path,
from:  --model_path gs://intelai_public_models/resnet_50_i8  
to : --model_path gs://public-artifacts/intelai_public_models/resnet_50_i8